app/models/base_organisation.rb

```diff
@@ -3,7 +3,17 @@ class BaseOrganisation < ActiveRecord::Base
   validates_url :website, :prefferred_scheme => 'http://', :if => Proc.new{|org| org.website.present?}
   validates_url :donation_info, :prefferred_scheme => 'http://', :if => Proc.new{|org| org.donation_info.present?}
   has_many :category_organisations, :foreign_key => :organisation_id
-  has_many :categories, :through => :category_organisations, :foreign_key => :organisation_id
+  has_many :categories, :through => :category_organisations, :foreign_key => :organisation_id do
+    def what_they_do
+      where(charity_commission_id: 100..199).order(name: :asc)
+    end
+    def who_they_help
+      where(charity_commission_id: 200..299).order(name: :asc)
+    end
+    def how_they_help
+      where(charity_commission_id: 300..399).order(name: :asc)
+    end
+  end
```

Comment:
I guess I'm not sure what we're trying to accomplish here. Are we trying to have scopes on organisations such as "give me all organisations that have categories that are `what_they_do`"? [Here's a commit to demo doing that.](https://github.com/johnnymo87/LocalSupport/commit/96a982a45092018f01511a693f4dcd21f029df8b) Note that I have to make organisations `DISTINCT` or else I'm getting the cartesian product, and I have to remove the `ORDER BY` from the category scope or else postgres boohoos about `categories.name` needing to be in the `SELECT`.

```
2.0.0-p598 :048 > Organisation.count
   (1.4ms)  SELECT COUNT(*) FROM "organisations" WHERE "organisations"."type" IN ('Organisation') AND "organisations"."deleted_at" IS NULL
 => 642
2.0.0-p598 :045 > Organisation.what_they_do.count
   (12.3ms)  SELECT DISTINCT COUNT(DISTINCT "organisations"."id") FROM "organisations" INNER JOIN "categories_organisations" ON "categories_organisations"."organisation_id" = "organisations"."id" INNER JOIN "categories" ON "categories"."id" = "categories_organisations"."category_id" WHERE "organisations"."type" IN ('Organisation') AND "organisations"."deleted_at" IS NULL AND ("categories"."charity_commission_id" BETWEEN 100 AND 199)
 => 631
2.0.0-p598 :046 > Organisation.how_they_help.count
   (13.7ms)  SELECT DISTINCT COUNT(DISTINCT "organisations"."id") FROM "organisations" INNER JOIN "categories_organisations" ON "categories_organisations"."organisation_id" = "organisations"."id" INNER JOIN "categories" ON "categories"."id" = "categories_organisations"."category_id" WHERE "organisations"."type" IN ('Organisation') AND "organisations"."deleted_at" IS NULL AND ("categories"."charity_commission_id" BETWEEN 300 AND 399)
 => 621
2.0.0-p598 :047 > Organisation.who_they_help.count
   (10.1ms)  SELECT DISTINCT COUNT(DISTINCT "organisations"."id") FROM "organisations" INNER JOIN "categories_organisations" ON "categories_organisations"."organisation_id" = "organisations"."id" INNER JOIN "categories" ON "categories"."id" = "categories_organisations"."category_id" WHERE "organisations"."type" IN ('Organisation') AND "organisations"."deleted_at" IS NULL AND ("categories"."charity_commission_id" BETWEEN 200 AND 299)
 => 632
```

Since almost all organisations are associated with categories of each subcategory, we get back almost all organisations from each scope. I'm not sure how useful this is ... but then again I haven't been following the story behind this PR. But if all we're trying to accomplish is `organisation.categories.how_they_help.pluck(:name).join(", ")`, then I don't think we need any of this scope merging stuff.